src/webui/src/static/function.ts

```diff
@@ -84,15 +84,18 @@ const getFinalResult = (final?: MetricDataRecord[]): number => {
     }
 };
 
+function isNaNorInfinity(val: number): boolean {
+    return Object.is(val, NaN) || Object.is(val, Infinity);
```

Comment:
I think `Number.isNaN` and `Number.isFinite` is more clear for this purpose.

Reply:
OK.